ssh-container/ssh-container.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: nnf.cray.hpe.com/v1alpha7
+kind: NnfContainerProfile
```

Comment:
I've forgotten how the uid/gid from the Workflow gets communicated to the user-container (oops). Do we have anything in here, maybe in the entrypoint.sh or somewhere else, where we confirm that the Workflow uid/gid matches the uid/gid baked into the image?

Reply:
If I'm clever enough to convince the admin to install my containerprofile which points at someone else's image, then that would be bad.

Reply:
Thanks for confirming.

Reply:
And even if you got past this part, ssh most likely wouldn't work unless you baked in the same public key.